performance/CMakeLists.txt

```diff
@@ -43,3 +47,9 @@ traccc_add_library( traccc_performance performance TYPE SHARED
    "src/performance/throughput.cpp" )
 target_link_libraries( traccc_performance
    PUBLIC ROOT::Core ROOT::RIO ROOT::Tree ROOT::Hist traccc::core )
+
+find_package(CUDAToolkit)
+if(CUDAToolkit_FOUND)
+        target_link_libraries( traccc_performance PRIVATE CUDA::cudart )
+                target_compile_options( traccc_performance PRIVATE )
```

Comment:
Please remove the `target_compile_options` line. It's not doing anything.

Reply:
Just removed this